pages/api/flatfile/save-custom-field.tsx

```diff
@@ -0,0 +1,39 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import { PrismaClient } from "@prisma/client";
+import { getToken } from "next-auth/jwt";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const token = await getToken({
+    req: req,
+  });
+  // console.log("gSSP token", token);
+
+  if (!token || !token.sub) {
+    console.log("No session");
+    return {
+      notFound: true,
+    };
+  }
+
+  const prisma = new PrismaClient();
```

Comment:
we should use the `prismaClient` in the `lib` folder to not connection spam ourselves